main_test.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"net/http"
+	"net/http/httptest"
+)
+
+var _ = Describe("cloud-native-go-helloworld-service", func() {
```

Comment:
I brought in Ginkgo and Gomega for more BDD-style tests with fluent assertions; but since the scope of these tests is pretty small, the standard `testing` package might make sense as well